jwst/tso_photometry/tso_photometry_step.py

```diff
@@ -5,112 +5,113 @@
 from ..lib.catalog_utils import replace_suffix_ext
 from .tso_photometry import tso_aperture_photometry
 
-__all__ = ['TSOPhotometryStep']
+__all__ = ["TSOPhotometryStep"]
 
 
 class TSOPhotometryStep(Step):
-    """
-    Perform circular aperture photometry on imaging Time Series
-    Observations (TSO).
-
-    Parameters
-    -----------
-    input : str or `CubeModel`
-        Filename for a FITS image or association table, or a `CubeModel`.
-    """
+    """Perform circular aperture photometry on imaging Time Series Observations (TSO)."""
 
     class_alias = "tso_photometry"
 
     spec = """
         save_catalog = boolean(default=False)  # save exposure-level catalog
-    """ # noqa: E501
+    """  # noqa: E501
 
-    reference_file_types = ['tsophot']
+    reference_file_types = ["tsophot"]
 
     def process(self, input_data):
-
+        """
+        Do the tso photometry processing.
+
+        Parameters
+        ----------
+        input_data : str or `CubeModel`
+            Filename for a FITS image or association table, or a `CubeModel`.
```

Comment:
```suggestion
            Filename pointing to a `CubeModel`, or a `CubeModel`.
```
Does this actually work with an association?

Reply:
IMO now would be a good time to check

Reply:
ok, good catch! I tested it and it does not work, I removed it.